recipes/pyjulia/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "pyjulia" %}
+{% set version = "0.5.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/julia-{{ version }}.tar.gz
+  sha256: 378d0377f75bb0e3bfc4cce19a56d3bf5a9a7be38e370e3a7cf3359bf4cd0378
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - julia-py = julia.julia_py:main
+    - python-jl = julia.python_jl:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - julia
+  run:
+    - python
+    - julia
```

```suggestion
    - julia >=1.1
```
Based on https://github.com/conda-forge/julia-feedstock/issues/83#issuecomment-624908506